cmd/octosql/main.go

```diff
@@ -77,23 +81,52 @@ With OctoSQL you don't need O(n) client tools or a large data analysis system de
 			log.Fatal(err)
 		}
 
-		var out output.Output
+		// stream json
+		// live csv
+		// live table
+		// batch csv
+		// batch table
+		var outputSinkFn func(stateStorage storage.Storage, streamID *execution.StreamID, eventTimeField octosql.VariableName) (execution.IntermediateRecordStore, output.Printer)
 		switch outputFormat {
```

Comment:
OK to be honest this is not very elegant, maybe consider if it's doable to put it into some subfunction?

Reply:
It won't really help in any way. And this is main, the place where you just construct all dependencies, and come with terms to how ugly it looks. I don't think putting this in a function will help with readability. It's also not used anywhere else.

I named the function type though.

Reply:
OK